meta-imx8mmsolidrun-support/conf/machine/include/lmp-factory-custom.inc

```diff
@@ -0,0 +1,20 @@
+# LMP factory specific customizations (either replace or extend options as defined by meta-lmp)
```

Comment:
Doesn't having this file lead to a situation where there are two lmp-factory-custom.inc files? One from here, and another from meta-lmp-support? Or is the idea to enable only one of them at time?

Reply:
Maybe this file should be removed. I'll do that.